### DISCUSSION TITLE
Möglichkeit der Berücksichtigung von header-Variablen bei url-redirection in der api.config

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#34] [Q&A]
Derzeit können header-Variablen in der api.config für die Definition z.B. der "outputUrl" verwendet werden.
z.B: {header:X-ORIG-SCHEME}://{header:X-ORIG-HOST}{header:X-ORIG-URI/}
Schön wäre es, könnte man diese auch für die url-redirection einsetzen.
Das ist vor allem zweckmäßig für die Definition von Portal-redirections.

**Comment by gis-eni** [comment]
Ab Build 5.23.901
